experiments/download_landsat.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import argparse
+import numpy as np
+import os
+import ee
+
+from geeS2downloader import GEES2Downloader
```

Comment:
Filtering of the cloud_pct was wrong, if I tried filering with `.filterMetadata` it would always yield 0, but  
```
.filter(
   ee.Filter.And(
        ee.Filter.gte(args.meta_cloud_name, 0),
        ee.Filter.lte(args.meta_cloud_name, args.cloud_pct),
   )
)
```
works.